testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/security/loginmodules/GenerateLDAPSJKSStore.java

```diff
@@ -52,6 +56,8 @@ private static SelfSignedX509CertificateAndSigningKey createSelfSigned() {
                 .setDn(new X500Principal(DN))
                 .setKeyAlgorithmName("RSA")
                 .setSignatureAlgorithmName(SHA_1_RSA)
+                .addExtension(new SubjectAlternativeNamesExtension(false, Collections.singletonList(
```

Comment:
Can customers have similar code in their staging or production environment?
If yes, I presume they will need to do similar changes to the their code. Is it correct assumption?

Reply:
This API was added to WildFly Elytron 1.2.0 so is not presently included in any supported releases.

Reply:
Thanks for clarification

Reply:
@rsvoboda They cannot have such code (it is code generating testing certificates), but they can have such X509 certificates from their CA/self-generated. However it is out of scope for AS to mention certificates generated by CAs should have alternative names extension for the IP.